### PR TITLE
GCW-2145 Require user to request at least one thing for book-appointment

### PR DESCRIPTION
**Pull request by abhinav098**
Hello Team, this PR will fix the issue for No Good request submission issue on goods_details page.
=> require user to request at least one thing 
Please review and comment.

### CODE REVIEW

File: app/controllers/order/goods_details.js [hunk 1]
```diff
@@ -11,6 +11,10 @@ export default Ember.Controller.extend({
   otherDetails: "",
   sortProperties: ["id"],
   sortedGcRequests: Ember.computed.sort("model.goodcityRequests", "sortProperties"),
+  gcRequestNull: Ember.computed("model.goodcityRequests", function() {
```
Comment (Darshan-Chauhan) [inline app/controllers/order/goods_details.js]: May be rename to hasGcRequest?

Also correct Indentation

File: app/controllers/order/goods_details.js [hunk 2]
```diff
@@ -11,6 +11,10 @@ export default Ember.Controller.extend({
...
+                    return (this.get('model.goodcityRequests').length === 0);
```
Comment (Darshan-Chauhan) [inline app/controllers/order/goods_details.js]: No need to compare with zero.
!this.get('model.goodcityRequests') should work

File: app/controllers/order/goods_details.js (hunk 1 above)
Comment (abhinav098) [reply app/controllers/order/goods_details.js]: Already done, was committing same.

File: app/controllers/order/goods_details.js (hunk 2 above)
Comment (abhinav098) [reply app/controllers/order/goods_details.js]: done, thanks :+1:

**Pull request merged**